doc/phpactor.txt

```diff
@@ -22,17 +23,14 @@ website:
 
 ================================================================================
                                                             *phpactor-installation*
-Install Phpactor using your favorite VIM package manager, I am using Vundle.
+Install Phpactor using your favorite VIM package manager.
+If you don't use any, I recommend vim-plug.
 Add the plugin to your .vimrc:
 
-`Plugin 'phpactor/phpactor'`
+`Plug 'phpactor/phpactor', {'for': 'php', 'do': composer install'}`
 
 Then in VIM:
-`:VundleInstall`
-
-You will need to install the Phpactor dependencies with composer:
-`$ cd ~/.vim/bundle/phpactor`
-`$ composer install`
```

Comment:
Leave this for those who are not using `vim-plug` (with appropriate note)

Reply:
Fixed it, thanks.